internal/scheduler/testfixtures/testfixtures.go

```diff
@@ -446,31 +395,31 @@ func GpuJob(queue string, priorityClassName string) *jobdb.Job {
 	)
 }
 
-func TestNSmallCpuJob(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
+func N1CpuPodReqs(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
 	rv := make([]*schedulerobjects.PodRequirements, n)
 	for i := 0; i < n; i++ {
-		rv[i] = TestSmallCpuJob(queue, priority)
+		rv[i] = Test1CpuPodReqs(queue, util.ULID(), priority)
 	}
 	return rv
 }
 
-func TestNLargeCpuJob(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
+func N32CpuPodReqs(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
 	rv := make([]*schedulerobjects.PodRequirements, n)
 	for i := 0; i < n; i++ {
-		rv[i] = TestLargeCpuJob(queue, priority)
+		rv[i] = Test32CpuPodReqs(queue, util.ULID(), priority)
 	}
 	return rv
 }
 
-func TestNGpuJob(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
+func N1GpuPodReqs(queue string, priority int32, n int) []*schedulerobjects.PodRequirements {
 	rv := make([]*schedulerobjects.PodRequirements, n)
 	for i := 0; i < n; i++ {
-		rv[i] = TestGpuJob(queue, priority)
+		rv[i] = Test1GpuPodReqs(queue, util.ULID(), priority)
 	}
 	return rv
 }
 
-func TestSmallCpuJob(queue string, priority int32) *schedulerobjects.PodRequirements {
+func Test1CpuPodReqs(queue string, jobId ulid.ULID, priority int32) *schedulerobjects.PodRequirements {
```

Comment:
The `jobId` argument is of type `ulid.ULID` instead of `string` so that it's harder to confuse it with `queue`.